swift/Sources/FlatBuffers/FlatBufferBuilder.swift

```diff
@@ -287,7 +287,11 @@ public struct FlatBufferBuilder {
     var isAlreadyAdded: Int?
 
     let vt2 = _bb.memory.advanced(by: _bb.writerIndex)
+#if swift(<5.7)
     let len2 = vt2.load(fromByteOffset: 0, as: Int16.self)
+#else
+    let len2 = vt2.loadUnaligned(fromByteOffset: 0, as: Int16.self)
+#endif
```

Comment:
nit: We don't need this. Since we are always sure that the builder has a properly aligned memory layout.

Reply:
Sure, if it's not needed there is not reason to have it here. Removed.